.github/workflows/loadtest.yaml

```diff
@@ -0,0 +1,45 @@
+name: Performance Test
+on: 
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+      tags:
+        description: 'K6 Load Test'  
+
+jobs:
+  k6_test_run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          repository: longwuyuan/ingress-nginx
+          ref: k6
+
+      - name: Make dev-env
+        run: |
+          mkdir $HOME/.kube
+          make dev-env
+          sleep 60
```

Comment:
Instead of using sleep, use kubectl wait for a condition of ingress-controller as an example :)

Reply:
Changed to kubectl wait. Please check